### PR TITLE
Docs for Teams monitoring and alerting capabilities 

### PR DESCRIPTION
**Pull request by ivlists**
Docs for Teams monitoring and alerting capabilities

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-alerts/device-health-status.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: Microsoft Teams Devices Monitoring and Alerting
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:2]:
```suggestion
title: Microsoft Teams devices monitoring and alerting
```

File: Teams/teams-alerts/device-health-status.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+ms.topic: article
+ms.service: msteams
+ms.reviewer: vapati
+f1.keywords:
+- 
+localization_priority: 
+search.appverid: 
+ms.collection: 
+  - 
+description: Learn how to use the Teams monitoring and alerting capabilities in the Microsoft Teams admin center to pro-actively monitor health state of Teams devices
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:16]:
```suggestion
description: Learn how to use the Teams monitoring and alerting capabilities in the Microsoft Teams admin center to pro-actively monitor the health state of Teams devices.
```

File: Teams/teams-alerts/device-health-status.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+appliesto: 
+  - Microsoft Teams
+ms.custom: 
+---
+# Microsoft Teams device health monitoring
+
+The device health monitoring in the Microsoft Teams admin center gives you an ability to pro-actively monitor health of various Teams devices. With an ability to monitor 'offline' state of a device,  you can get alerted in real-time if the monitored device in your organization goes offline.
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:23]:
```suggestion
The device health monitoring in the Microsoft Teams admin center gives you an ability to pro-actively monitor the health of various Teams devices. With an ability to monitor the 'offline' state of a device,  you can get alerted in real-time if the monitored device in your organization goes offline.  
```

File: Teams/teams-alerts/device-health-status.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+description: Learn how to use the Teams monitoring and alerting capabilities in the Microsoft Teams admin center to pro-actively monitor health state of Teams devices
+appliesto: 
+  - Microsoft Teams
+ms.custom: 
+---
+# Microsoft Teams device health monitoring
+
+The device health monitoring in the Microsoft Teams admin center gives you an ability to pro-actively monitor health of various Teams devices. With an ability to monitor 'offline' state of a device,  you can get alerted in real-time if the monitored device in your organization goes offline.  
+
+> **Pre-requisite :**
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:25]:
```suggestion
> **Prerequisite :**
```

File: Teams/teams-alerts/device-health-status.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+> An admin who is configuring this rule should have a Team/channel creation permissions in his tenant. [Learn More](https://docs.microsoft.com/en-us/microsoft-365/solutions/manage-creation-of-groups?view=o365-worldwide)
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:26]:
```suggestion
> An admin who is configuring this rule should have Teams/channel creation permissions in his tenant. For more information, see [Manage who can create Microsoft 365 Groups
](https://docs.microsoft.com/microsoft-365/solutions/manage-creation-of-groups?view=o365-worldwide).
```

File: Teams/teams-alerts/device-health-status.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+
+## Configure device state rule
+
+1. In the left navigation of the Microsoft Teams admin center, click **Notifications & alerts** > **Rules**. On the **Rules** Page, select **Device state rule**.
+
+
+    ![Screenshot of the rules page in the Teams admin center with differnt out of box rules](../media/teams-alert-rules-page.png "Screenshot of the Teams alerting rules in the Teams admin center")
+    
+2. Once clicked, Device state rule needs to configure in detail before enabling it for alerts. Please follow below guide to interprete each section.
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:36]:
```suggestion
2. Once selected, the **Device state rule** needs to be configured in detail before enabling it for alerts. Please see the following guidance to interpret each section. 
```

**Comment by ivlists** [comment]
@MicrosoftHeidi  I have added the copy edits suggested by you. Please feel free to merge the branch.

File: Teams/teams-alerts/device-health-status.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+appliesto: 
+  - Microsoft Teams
+ms.custom: 
+---
+# Microsoft Teams device health monitoring
+
+Device health monitoring in the Microsoft Teams admin center gives you an ability to proactively monitor the health of various Teams devices. With an ability to monitor the 'offline' state of a device,  you can get alerted in real-time if the monitored device in your organization goes offline.  
+
+> **Prerequisite :**
+> An admin configuring this rule should have teams/channel creation permissions in his tenant. [Learn More](https://docs.microsoft.com/en-us/microsoft-365/solutions/manage-creation-of-groups?view=o365-worldwide)
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:26]:
```suggestion
> An admin configuring this rule should have teams/channel creation permissions in his tenant. To learn more, see 
[Manage who can create Microsoft 365 Groups](https://docs.microsoft.com/microsoft-365/solutions/manage-creation-of-groups?view=o365-worldwide).
```

File: Teams/teams-alerts/device-health-status.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+    ![Screenshot of the rules page in the Teams admin center with different out of the box rules.](../media/teams-alert-rules-page.png "Screenshot of the Teams alerting rules in the Teams admin center")
+    
+2. Once clicked, the device state rule needs to be configured in detail before enabling it for alerts. Please follow the below guide to interpret each section. 
+
+    ![Screenshot of the Teams devices state rule in the Teams admin center.](../media/teams-device-health-state-alerts-with-callouts.png "Screenshot of the Device state rule in the Teams admin center.")
+
+## Interpret the rule configuration
+|Callout |Description  |
+|--------|-------------|
+|**1**   |The device state rule helps you effectively manage Teams devices and is classified as a device management type. In the future, more rules of device management type will be available to monitor other related capabilities (examples may include: unhealthy device, Sign-in status of device).|
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:43]:
```suggestion
|**1**   |The device state rule helps you effectively manage Teams devices and is classified as a device management type. In the future, more rules of a device management type will be available to monitor other related capabilities (examples may include: unhealthy device, Sign-in status of device).|
```

File: Teams/teams-alerts/device-health-status.md [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+|**2**   |You can monitor the health of devices if they go offline. [Learn more](https://docs.microsoft.com/en-us/microsoftteams/devices/device-management) about device management in Teams admin center. |
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:44]:
```suggestion
|**2**   |You can monitor the health of devices if they go offline. To learn more about device management in the Teams admin center, see [Manage your devices in Microsoft Teams](https://docs.microsoft.com/microsoftteams/devices/device-management) . |
```

File: Teams/teams-alerts/device-health-status.md [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+|**3**   |You can specify how frequently you want to monitor device health status by mentioning the rule evaluation frequency. By default teams devices will be monitored in near real-time if they go offline. |
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:45]:
```suggestion
|**3**   |You can specify how frequently you want to monitor device health status by mentioning the rule evaluation frequency. By default, Teams devices will be monitored in near real-time if they go offline. |
```

File: Teams/teams-alerts/device-health-status.md [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+|**4**   |You can specify which devices needs proactive offline statue monitoring by selecting them based on signed-in users. Please refer to [Select devices for configuration](#Select-devices-for-configuration) for more details. |
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:46]:
```suggestion
|**4**   |You can specify which devices need proactive offline statue monitoring by selecting them based on signed-in users. Please refer to [Select devices for configuration](#Select-devices-for-configuration) for more details. |
```

File: Teams/teams-alerts/device-health-status.md [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+|**5**   |In the Actions section, you can specify teams channels you want to get alerts for. Currently, a default team named **'Admin Alerts and Notifications'** and channel named **'MonitoringAlerts'** will be created where notifications will be delivered to. <BR/> <BR/> Global administrators and Teams administrators in your tenant will be automatically added to this default team.|
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:47]:
```suggestion
|**5**   |In the **Actions** section, you can specify Teams channels you want to get alerts for. Currently, a default team named **Admin Alerts and Notifications** and a channel named **MonitoringAlerts** will be created where notifications will be delivered to. <BR/> <BR/> Global administrators and Teams administrators in your tenant will be automatically added to this default team.|
```

File: Teams/teams-alerts/device-health-status.md [hunk 13]
```diff
@@ -0,0 +1,85 @@
...
+ 
+
+
+## Select devices for configuration
+
+1. You can select Teams devices you want to monitor by selecting users signed in to those devices. Click on **Add** in **Device users** section
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:54]:
```suggestion
1. You can select Teams devices you want to monitor by selecting users signed in to those devices. Click on **Add** in the **Device users** section.
```

File: Teams/teams-alerts/device-health-status.md [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+
+    ![Screenshot of the add user in device health status rule.](../media/add-users-device-state-rule.png "Screenshot of the add users in device status rule.")
+
+2. Select one or more users for which you want to monitor device health state
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:58]:
```suggestion
2. Select one or more users for which you want to monitor the device health state.
```

File: Teams/teams-alerts/device-health-status.md [hunk 15]
```diff
@@ -0,0 +1,85 @@
...
+
+    ![Screenshot of the add user in device health status rule.](../media/add-users-device-state-rule.png "Screenshot of the add users in device status rule.")
+
+2. Select one or more users for which you want to monitor device health state 
+
+    ![Screenshot of the add user in device health status rule.](../media/add-users-in-device-rule.png "Screenshot of the add users in device status rule.")
+
+    ![Screenshot of the add user in device health status rule.](../media/add-users-in-device-rule-2.png "Screenshot of the add users in device status rule.")
+
+3. Selected list of users will be shows in **Device users** section. You can modify this list by adding or removing users.
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:64]:
```suggestion
3. The selected list of users will be shown in the **Device users** section. You can modify this list by adding or removing users.
```

File: Teams/teams-alerts/device-health-status.md [hunk 16]
```diff
@@ -0,0 +1,85 @@
...
+
+    ![Screenshot of the add user in device health status rule.](../media/added-users-in-device-rule.png "Screenshot of the add users in device status rule.")
+
+4. All the sign-in devices used by the selected list of users will be monitored for the offline health state.
+
+## Notifications in Teams client
+
+The notifications will be delivered in the auto-created **MonitoringAlerts** channel of the **Admin Alerts and Notifications** Team.
+
+Sample device offline notifications :
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:74]:
```suggestion
Sample device offline notifications: 
```

File: Teams/teams-alerts/device-health-status.md [hunk 17]
```diff
@@ -0,0 +1,85 @@
...
+Sample device offline notifications : 
+    
+   ![Screenshot of the device offline alert in Teams client.](../media/device-offline-alert-with-callouts.png "Screenshot of the device offline alert in MonitoringAlerts channel in Teams.")
+
+## Interpret the device offline notifications 
+|Callout |Description  |
+|--------|-------------|
+|**1**   |The device name that's offline. |
+|**2**   |The user of the offline device. |
+|**3**   |What time the device went offline. (Currently, the time is presented in UTC) |
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:83]:
```suggestion
|**3**   |What time the device went offline. (Currently, the time is presented in UTC.) |
```

File: Teams/teams-alerts/teams-admin-alerts.md [hunk 18]
```diff
@@ -0,0 +1,47 @@
...
+audience: Admin
+ms.topic: conceptual
+ms.service: msteams
+ms.reviewer: vapati
+f1.keywords:
+localization_priority: Normal
+search.appverid:
+ms.collection: 
+  - M365-collaboration
+description: In this article, you will learn about the Teams alerts and nitifications capabilities avalbale in the Microsoft Teams admin center.
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/teams-admin-alerts.md:15]:
```suggestion
description: In this article, you will learn about the Teams alerts and notifications capabilities available in the Microsoft Teams admin center.
```

File: Teams/teams-alerts/teams-admin-alerts.md [hunk 19]
```diff
@@ -0,0 +1,47 @@
+---
+title: Microsoft Teams Monitoring and Alerting
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/teams-admin-alerts.md:2]:
```suggestion
title: Microsoft Teams monitoring and alerting 
```

File: Teams/teams-alerts/device-health-status.md [hunk 20]
```diff
@@ -0,0 +1,85 @@
+---
+title: Microsoft Teams Devices Monitoring and Alerting
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/device-health-status.md:2]:
```suggestion
title: Microsoft Teams devices monitoring and alerting
```

File: Teams/teams-alerts/teams-admin-alerts.md [hunk 21]
```diff
@@ -0,0 +1,47 @@
...
+search.appverid:
+ms.collection: 
+  - M365-collaboration
+description: In this article, you will learn about the Teams alerts and nitifications capabilities avalbale in the Microsoft Teams admin center.
+appliesto: 
+  - Microsoft Teams
+ms.custom: 
+---
+
+# Microsoft Teams Monitoring and Alerting
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/teams-admin-alerts.md:21]:
```suggestion
# Microsoft Teams monitoring and alerting
```

File: Teams/teams-alerts/teams-admin-alerts.md [hunk 22]
```diff
@@ -0,0 +1,47 @@
...
+
+New monitoring and alerting capabilities for Microsoft Teams are available in the Microsoft Teams admin center. You can use different sets of rules availble under 'Notifications and Rules' section in Teams admin center to proactively monitor various Teams capablities, and you can be alerted if anything goes wrong with them. For example, you can actively monitor the health of Teams devices (IP Phones, collaboration bars, etc.) if they unexpectedly go offline.
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/teams-admin-alerts.md:23]:
```suggestion
New monitoring and alerting capabilities for Microsoft Teams are available in the Microsoft Teams admin center. You can use different sets of rules available under the **Notifications and Rules** section in the Teams admin center to proactively monitor various Teams capabilities, and you can be alerted if anything goes wrong with them. For example, you can actively monitor the health of Teams devices (such as IP phones and collaboration bars) if they unexpectedly go offline.  
```

File: Teams/teams-alerts/teams-admin-alerts.md [hunk 23]
```diff
@@ -0,0 +1,47 @@
...
+appliesto: 
+  - Microsoft Teams
+ms.custom: 
+---
+
+# Microsoft Teams Monitoring and Alerting
+
+New monitoring and alerting capabilities for Microsoft Teams are available in the Microsoft Teams admin center. You can use different sets of rules availble under 'Notifications and Rules' section in Teams admin center to proactively monitor various Teams capablities, and you can be alerted if anything goes wrong with them. For example, you can actively monitor the health of Teams devices (IP Phones, collaboration bars, etc.) if they unexpectedly go offline.  
+
+Your organization can use Teams monitoring and alerting to automatically manage Teams capabilities, be alerted if they show something unexpected, and take corrective actions to get things back on-track.
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/teams-admin-alerts.md:25]:
```suggestion
Your organization can use Teams monitoring and alerting to automatically manage Teams capabilities, be alerted if they show something unexpected, and take corrective actions to get things back on track.
```

File: Teams/teams-alerts/teams-admin-alerts.md [hunk 24]
```diff
@@ -0,0 +1,47 @@
...
+
+## How to manage Monitoring and Alerting
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/teams-admin-alerts.md:27]:
```suggestion
## How to manage monitoring and alerting
```

File: Teams/teams-alerts/teams-admin-alerts.md [hunk 25]
```diff
@@ -0,0 +1,47 @@
...
+
+To configure alerting rules, you must be a global admin in Microsoft 365 or a Teams service admin. To learn more about Teams admin roles and which reports each admin role can access, see [Use Teams administrator roles to manage Teams](../using-admin-roles.md).
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/teams-admin-alerts.md:29]:
```suggestion
To configure alerting rules, you must be a Global admin in Microsoft 365 or a Teams service admin. To learn more about Teams admin roles and which reports each admin role can access, see [Use Teams administrator roles to manage Teams](../using-admin-roles.md).
```

File: Teams/teams-alerts/teams-admin-alerts.md [hunk 26]
```diff
@@ -0,0 +1,47 @@
...
+
+Go to the Microsoft Teams admin center, and in the left navigation select **Notifications & alerts**. Under **Rules** choose the rule you want to configure.
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/teams-admin-alerts.md:31]:
```suggestion
Go to the Microsoft Teams admin center, and in the left navigation pane, select **Notifications & alerts**. Under **Rules**, choose the rule you want to configure.
```

File: Teams/teams-alerts/teams-admin-alerts.md [hunk 27]
```diff
@@ -0,0 +1,47 @@
...
+
+
+
+## Teams monitoring rules reference
+
+HHere's a list of the Teams monitoring rules available in the Microsoft Teams admin center.
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/teams-admin-alerts.md:37]:
```suggestion
The following is a list of the Teams monitoring rules available in the Microsoft Teams admin center.
```

File: Teams/teams-alerts/teams-admin-alerts.md [hunk 28]
```diff
@@ -0,0 +1,47 @@
...
+
+We're continually adding and improving the Teams monitoring experience by adding various monitoring capabilities and configuration functionalities. 
+
+|Rule  |Monitoring capability|What's monitored? |
+|---------|---------|---------|
+|[Device health status](device-health-status.md)  |Teams Devices | Pro-actively monitor Teams devices if they goes offline
```
Comment (JohanFreelancer9) [inline Teams/teams-alerts/teams-admin-alerts.md:43]:
```suggestion
|[Device health status](device-health-status.md)  |Teams Devices | Pro-actively monitors Teams devices if they go offline.
```

File: Teams/using-admin-roles.md [hunk 29]
```diff
@@ -39,7 +39,7 @@ To follow along, you must be an admin. The instructions for getting the permissi
...
...ent.<sup>1</sup></li><li>Manage messaging, including messaging policies.<sup>1,3</sup></li><li>Manage all org-wide settings, including federation, teams upgrade, and teams client settings.<sup>1,3</sup></li><li>Manage the teams in the organization and their associated settings, including membership (group management supported via PowerShell, team management in the Teams admin center).<sup>2,3</sup></li><li>Manage Teams-certified devices and set up and assign configuration policies.<sup>2</sup></li><li>View user profile page and troubleshoot user call quality problems using advanced troubleshooting toolset.<sup>3</sup> </li><li>Access all reports in the Microsoft Teams admin center</li><li>Monitor and configure alerts for various teams capabilities (Teams devices health etc. )</li><li> Access, monitor and troubleshoot tenant's call quality and reliability using data exposed in Call Quality Dashboard (CQD) down to the users impacted by poor call quality. Create new call quality reports, update and remove call quality reports as needed. Upload and update CQD building data.</li><li> [Publish apps to the tenant app catalog in the Microsoft Teams admin center](manage-apps.md)</li></ul> |
```
Comment (JohanFreelancer9) [inline Teams/using-admin-roles.md:42]:
```suggestion
| Teams Service Administrator             | Manage the Teams service, and manage and create Microsoft 365 Groups.        | Everything in the Microsoft Teams admin center and associated PowerShell controls, including:<ul><li> Manage meetings, including meeting policies, configurations, and conference bridges.<sup>1,3</sup></li><li>Manage voice, including calling policies and phone number inventory and assignment.<sup>1</sup></li><li>Manage messaging, including messaging policies.<sup>1,3</sup></li><li>Manage all org-wide settings, including federation, teams upgrade, and teams client settings.<sup>1,3</sup></li><li>Manage the teams in the organization and their associated settings, including membership (group management supported via PowerShell, team management in the Teams admin center).<sup>2,3</sup></li><li>Manage Teams-certified devices and set up and assign configuration policies.<sup>2</sup></li><li>View user profile page and troubleshoot user call quality problems using advanced troubleshooting toolset.<sup>3</sup> </li><li>Access all reports in the Microsoft Teams admin center</li><li>Monitor and configure alerts for various teams capabilities (such as Teams devices health)</li><li> Access, monitor and troubleshoot tenant's call quality and reliability using data exposed in Call Quality Dashboard (CQD) down to the users impacted by poor call quality. Create new call quality reports, update and remove call quality reports as needed. Upload and update CQD building data.</li><li> [Publish apps to the tenant app catalog in the Microsoft Teams admin center](manage-apps.md)</li></ul> |
```

Comment (JohanFreelancer9) [reply Teams/using-admin-roles.md:42]: Please change "Teams devices health etc." to "such as Teams devices health".

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**